addons/crm/tests/test_crm_pls.py

```diff
@@ -389,3 +390,44 @@ def test_predictive_lead_scoring(self):
         self.assertEqual(tools.float_compare(lead_tag_1.automated_probability, 28.6, 2), 0)
         self.assertEqual(tools.float_compare(lead_tag_2.automated_probability, 28.6, 2), 0)
         self.assertEqual(tools.float_compare(lead_tag_1_2.automated_probability, 28.6, 2), 0)
+
+        # C. Test Update Probabilities Wizard from the settings
```

Comment:
I would move them to a specific test method ... unit test should be as unit as possible :D .